src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -264,6 +276,11 @@
                                 </Grid>
                             </Expander>
                         </Border>
+                        <DataTemplate.Triggers>
+                            <DataTrigger Binding="{Binding Path=Unloaded}" Value="true">
+                                <Setter TargetName="PackageGrid" Property="Opacity" Value="0.5" />
```

Comment:
What is this for? Was this in the UI Spec?

Reply:
Ugh..I do not remember why I thought this was a good idea.
Basically makes Unloaded packages to appear greyed out

I will attach a snapshot. Maybe @Jingyi-Wen can tell us if it is useful or not

Reply:
![image](https://user-images.githubusercontent.com/46732933/128885140-0425d9d1-90b2-46c3-bd30-74bba1bd2db3.png)

Reply:
I can see this being helpful in terms of making the state more distinguishable. However in this case it may also feel like the context menu is disabled, which could be confusing